### PR TITLE
set up components for Header, NavContainer, ArticleCards and Footer, …

### PR DESCRIPTION
**Pull request by ELR143**
…create resuable Card component for styling, create ArticleCards, clean up files

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,35 +1,18 @@
...
-import './App.css'
+import { useState } from "react";
+import React from 'react'
+import "./App.css";
+import Header from "./components/Header";
+import ArticleCards from "./components/ArticleCards";
 
 function App() {
-  const [count, setCount] = useState(0)
+const [articles, setArticles] = useState([])
```
Comment (popatre) [inline src/App.jsx]: Does this need to live in the app?

File: src/components/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { useState } from "react";
+
+export function Card({ children }) {
+  const [isClicked, setIsClicked] = useState(false);
+
+  function handleClick() {
+    console.log('clicked')
```
Comment (popatre) [inline src/components/Card.jsx]: remove log

**Review by popatre — CHANGES_REQUESTED** [review]
- Excellent. Can view all the articles in a clear list
- Nice job splitting out your axios/fetch requests into their own api file, makes it much easier to maintain them here

- [x] Please add an is loading pattern to inform the user that something is happening, especially if the Internet connection is slow, or the server is slow to respond

**Review by nc-poppy-m — APPROVED** [review]
Good work on your loading message and on placement of state! Ready to merge

**Pull request merged**